VMEncryption/main/DiskUtil.py

```diff
@@ -808,7 +811,8 @@ def is_data_disk(self, device_item, special_azure_devices_to_skip):
         # IDE devices (in Gen 1) include Resource disk and BEK VOLUME. This check works pretty wel in Gen 1, but not in Gen 2.
         for azure_blk_item in special_azure_devices_to_skip:
             if azure_blk_item.name == device_item.name:
-                self.logger.log(msg="{0} is one of special azure devices that should be not considered data disks.".format(device_item.name))
+                if device_item.name:
```

Comment:
is this needed? if so add a comment

Reply:
leaving - azure_blk_item and device_item are both caller controlled, this is a safeguard against invalid input